index.md

```diff
@@ -31,7 +31,7 @@ Log in to [GitHub](http://github.org) and fork this [repository](https://github.
 
 ### Look at it
 
-You should write all your code in `app.js`.
+You should write all your code in `app.js`. The `support.js` file contains some helper functions that has been written for you, don't edit this file! All the HTML & CSS are supplied.
```

Comment:
The `support.js` file contains some helper functions that has been written for you, don't edit this file! All the HTML & CSS are supplied.

->

The `support.js` file contains some helper functions that have been written for you. Don't edit this file! All the HTML & CSS is supplied: don't edit these either!